Library/Homebrew/cmd/pull.rb

```diff
@@ -320,6 +320,7 @@ def files_changed_in_patch(patchfile, tap)
   def current_versions_from_info_external(formula_name)
     versions = {}
     json = Utils.popen_read(HOMEBREW_BREW_FILE, "info", "--json=v1", formula_name)
+    json.force_encoding("UTF-8")
```

Comment:
We usually just ignore this for Ruby 1.8. See https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/formulary.rb#L38-L46